azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+- master
+
+
+jobs:
+- job: Test
+  strategy:
+    matrix:
+      node_12: 
+        node_version: '12'
+        TEST_REPOSITORY: 'CORE'
+      node_10:
+        node_version: '10'
+        TEST_REPOSITORY: 'CORE'
+      node_8:
+        node_version: '8'
+        TEST_REPOSITORY: 'CORE'
+  variables:
+    TEST_REPOSITORY: $(test_repository)
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: $(node_version)
+      displayName: 'Install Node.js'
+    - script: |
+        yarn
+      displayName: 'Install dependencies'
+    - script: |
+        yarn run ci
+      displayName: 'Run CI'
+
+- job: Extended_tests
```

Comment:
"extended_tests" --> "e2e tests" ?

Reply:
I will rename it "e2e_tests" since we can add spaces on the job name